.github/workflows/build.yml

```
@@ -227,7 +227,102 @@ jobs:
           rustup component add clippy
       - name: Run default clippy linting
         run: |
-          cargo clippy -- -Aclippy::erasing_op -Aclippy::never_loop -Aclippy::if_same_then_else -Dclippy::try_err
+          cargo clippy -- \
```

Should we deny warnings?
```suggestion
          RUSTFLAGS='-D warnings' cargo clippy -- \
```

---

Right, this is run on stable... We'll see how that goes then, but probably a good idea to deny warnings from the start.

---

Done, I denied warnings by default.